tools/ConvertTo-PseudoLocalization.ps1

```diff
@@ -98,7 +95,7 @@ function GetPseudoLocalization([string]$key, [string]$value, [string]$comment) {
     $hash = [System.BitConverter]::ToInt32($hash)
     $rng = [System.Random]::new($hash)
 
-    $lines = $value.Split("`n")
+    $lines = $value -split '\r?\n'

```

Comment:
Some files seem to be LF? So that failed to work for some files which I didn't notice.